.github/workflows/gclientgen.yaml

```diff
@@ -0,0 +1,75 @@
+name: Client Generator
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        go-version: [ '1.23' ]
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Setup Go ${{ matrix.go-version }}
+      uses: actions/setup-go@v5
+      with:
+        go-version: ${{ matrix.go-version }}
+        cache-dependency-path: |
+          go.work.sum
+          gclientgen/go.sum
+    - name: Display Go version
+      run: go version
+    - run: go mod download
+    - run: go -C gclientgen test ./...
+
+  lint:
+    runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        go-version: [ '1.23' ]
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Setup Go ${{ matrix.go-version }}
+      uses: actions/setup-go@v5
+      with:
+        go-version: ${{ matrix.go-version }}
+        cache-dependency-path: |
+          go.work.sum
+          gclientgen/go.sum
+    - name: Display Go version
+      run: go version
+    - run: go -C gclientgen mod tidy
+    - run: go mod download
```

Comment:
just `go mod tidy` should be sufficient (you don't need `go mod download`)

Reply:
Done.